plugins/render/api_key.go

```diff
@@ -0,0 +1,82 @@
+package render
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Profiles          map[string]Profile
+}
+
+type Profile struct {
+	APIKey        string `yaml:"apiKey"`
+}
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://render.com/docs"),
+		ManagementURL: sdk.URL("https://dashboard.render.com/u/settings#api-keys"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.APIKey,
+				MarkdownDescription: "API Key used to authenticate to Render.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 30,
+					Charset: schema.Charset{
+						Uppercase: true,
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.TempFile(renderConfig, provision.AtFixedPath(ConfigPath())),
+		Importer: importer.TryAll(
+			TryRenderConfigFile(),
+		),
+	}
+}
+
+// var defaultEnvVarMapping = map[string]sdk.FieldName{
+// }
+func renderConfig(in sdk.ProvisionInput) ([]byte, error) {
+	config := Config{
+
+	}
+	contents, err := yaml.Marshal(&config)
+	if err != nil {
+		return nil, err
+	}
+	return []byte(contents), nil
+}
+
+func TryRenderConfigFile() sdk.Importer {
+	return importer.TryFile(".render/config.yaml", func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		var config Config
+		if err := contents.ToYAML(&config); err != nil {
+			out.AddError(err)
+			return
+		}
+
+		profile, ok := config.Profiles["default"]
+		if !ok || profile.APIKey == "" {
+			return
+		}
+
+		out.AddCandidate(sdk.ImportCandidate{
+			Fields: map[sdk.FieldName]string{
+				fieldname.APIKey: profile.APIKey,
+			},
+		})
```

Comment:
Let's import all profiles, not only the default one, and use the profile name as a name hint in the import candidate.

Reply:
> Let's import all profiles, not only the default one, and use the profile name as a name hint in the import candidate.

I have updated this.